cosmo_tester/test_suites/ha/ha_cluster_scenarios_test.py

```diff
@@ -29,28 +29,28 @@
 
 
 @pytest.fixture(scope='function', params=[2, 3])
-def cluster(
+def hosts(
```

Comment:
same as the other - this actually is a cluster, so could've been just left as is